flow/testing/mock_embedder.h

```diff
@@ -0,0 +1,44 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLOW_TESTING_MOCK_EMBEDDER_H_
+#define FLOW_TESTING_MOCK_EMBEDDER_H_
+
+#include "flutter/flow/embedded_views.h"
+
+namespace flutter {
+namespace testing {
+
+class MockViewEmbedder : public ExternalViewEmbedder {
```

Comment:
It's in the same space where we define other Mock objects that help with layer_unittests like mock_layer and mock_raster_cache and mock_texture. Over time it can provide more testing insights, but its first need is just to be a standin for having the view_embedder field of the context to be non-null.